win32/devdoc/interlocked_hl_requirements.md

```diff
@@ -66,7 +66,7 @@ MOCKABLE_FUNCTION_WITH_RETURNS(, INTERLOCKED_HL_RESULT, InterlockedHL_WaitForVal
 
 **SRS_INTERLOCKED_HL_01_005: [** When waiting for the value at address to change, the `milliseconds` argument value shall be used as timeout. **]**
 
-**SRS_INTERLOCKED_HL_01_007: [** When `WaitOnAddress` succeeds, the value at address shall be compared to the target value passed in `value` by using `InterlockedAdd`. **]**
+**SRS_INTERLOCKED_HL_01_007: [** When `WaitOnAddress` succeeds, the value at address shall be compared to the target value passed in `value` by using `interlocked_add`. **]**
```

Comment:
>WaitOnAddress [](start = 39, length = 13)

wait_on_address here and everywhere

Reply:
Will be done as part of a subsequent PR.

---
In reply to: [499933310](https://github.com/Azure/azure-c-pal/pull/39#discussion_r499933310) [](ancestors = 499933310)